scan/src/reusable/InfoMobileCard.re

```diff
@@ -82,6 +88,17 @@ let make = (~info) => {
       <Text value=name ellipsis=true />
     </div>
   | RequestID(id) => <TypeID.Request id />
+  | RequestResponse({request, responseTime}) =>
+    <div className={CssHelper.flexBox()}>
+      <Text value={request |> Format.iPretty} weight=Text.Regular block=true ellipsis=true />
+      <HSpacing size=Spacing.sm />
+      <Text
+        value={"(" ++ (responseTime |> Format.iPretty) ++ "ms)"}
+        weight=Text.Regular
+        block=true
+        color=Colors.gray6
```

Comment:
If there are a lot of places to show response time in ms, we may create a component to render it.